src/main/java/pl/allegrotech/productsshop/api/ApiExceptionHandler.java

```diff
@@ -0,0 +1,30 @@
+package pl.allegrotech.productsshop.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class ApiExceptionHandler {
```

Comment:
opowiadaliśmy już o ty,?

Reply:
wspominaliśmy. Można dopowiedzieć jeszcze we wtorek, ale bym zostawił :)

Reply:
a zostawmy jeszcze jak ;)